go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/sns v1.26.7
 	github.com/aws/aws-sdk-go-v2/service/sqs v1.29.7
 	github.com/aws/aws-sdk-go-v2/service/sts v1.26.7
-	github.com/aws/aws-sigv4-auth-cassandra-gocql-driver-plugin v0.0.0-20220331165046-e4d000c0d6a6
+	github.com/aws/aws-sigv4-auth-cassandra-gocql-driver-plugin v1.1.0
```

Comment:
FYI @smallinsky @greedy52, this one is jumping from v0.x to v1.x. I don't see anything concerning, but I thought I would let you know regardless.

Reply:
I checked the diff and nothing major has changed so it should be fine. There is a new helper `sigv4.NewAwsAuthenticatorWithCredentialCallback` that we can use the make our implementation slightly more readable but it is not necessary.

Reply:
Thanks, Steve!